### DISCUSSION TITLE
Fichas de usuários para Pagamentos, como trabalhar com essa limitação?

### DISCUSSION THREAD

**Discussion by albertosilva1974** [bacen/pix-api#416] [General]
Atualmente, nosso PSP informa que temos 50 fichas e que a cada chave CPF/CNPJ não encontrada no DICT perdemos 2, ou seja, com 25 insucessos, não temos como enviar solicitações de pagamento para as próximas 2,5 horas (reposição e 1 ficha a cada 3 minutos).
Alguém enfrentando essa limitação com ideias para orquestrar os envios?
Alguma previsão do BACEN aumentar essa quantidade?

**Comment by rubenskuhl** [comment]
> Atualmente, nosso PSP informa que temos 50 fichas e que a cada chave CPF/CNPJ não encontrada no DICT perdemos 2, ou seja, com 25 insucessos, não temos como enviar solicitações de pagamento para as próximas 2,5 horas (reposição e 1 ficha a cada 3 minutos).
> Alguém enfrentando essa limitação com ideias para orquestrar os envios?
> Alguma previsão do BACEN aumentar essa quantidade?

Esse limitador é o que previne harvesting do DICT. Não tem pq ser aumentado. 

O que você pode fazer é se tornar uma iniciador de pagamento autorizado, e aí você passa a ter acesso direto ao DICT. O BACEN ainda está especificando essa intersecção do Pix com o Open Banking.
